### PR TITLE
Add readme source

### PR DESCRIPTION
**Pull request by mark-tate**

### CODE REVIEW

File: docs/configure/sources/storybook-figma.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ A Figma pattern tag is defined within a Figma project's `sharedPluginData`.
 (Refer to the JPM Figma plugin for more details - a current Q4 2023 deliverable)
 
 For each retrieved/tagged pattern, a page is created in the Mosaic file-system.
-Additional Mosaic medata can be added, to each created page, using `projects.meta`
+Additional Mosaic medata can be added, to each created page, using `meta`
```
Comment (DavieReid) [inline docs/configure/sources/storybook-figma.mdx:18]: metadata

File: packages/source-readme/src/__tests__/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+};
+
+const options = {
+  prefixDir: 'prefixDir',
+  accessToken: 'Bearer mytoken',
+  readme: [
+    {
+      name: 'some-page-name-1',
+      readmeUrl: 'https://some/url/for/readme1.md',
+      // proxyEndpoint: 'http://proxy.jpmchase.net:8443',
```
Comment (DavieReid) [inline packages/source-readme/src/__tests__/index.test.ts:20]: delete

File: packages/source-readme/src/__tests__/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      contentTemplate: '<div>::content::</div>',
+      meta: {
+        layout: 'DetailTechnical',
+        title: 'some title 1',
+        description: 'some description 1',
+        tags: ['some-tag-1'],
+        data: {
+          // image:
+          //   'https://jpmc-cib.adobecqms.net/jpmcp-cm/content/dam/jpm-cp/dp-visual-designs/quick-link-images/img_resolve_1x.png',
```
Comment (DavieReid) [inline packages/source-readme/src/__tests__/index.test.ts:29]: delete

File: packages/source-readme/src/__tests__/index.test.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+          repoUrl: 'https://some/repo/url/1'
+        }
+      }
+    },
+    {
+      name: 'some-page-name-2',
+      readmeUrl: 'https://some/url/for/readme2.md',
+      // proxyEndpoint: 'http://proxy.jpmchase.net:8443',
```
Comment (DavieReid) [inline packages/source-readme/src/__tests__/index.test.ts:37]: delete

File: packages/source-readme/src/__tests__/index.test.ts [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+      contentTemplate: '<div>::content::</div>',
+      meta: {
+        layout: 'DetailTechnical',
+        title: 'some title 2',
+        description: 'some description 2',
+        tags: ['some-tag-2'],
+        data: {
+          // image:
+          //   'https://jpmc-cib.adobecqms.net/jpmcp-cm/content/dam/jpm-cp/dp-visual-designs/quick-link-images/img_resolve_1x.png',
```
Comment (DavieReid) [inline packages/source-readme/src/__tests__/index.test.ts:46]: delete

File: packages/source-readme/package.json
Comment (DavieReid) [inline packages/source-readme/package.json:1]: latest beta version is 59 and should be used for all mosaic imports and the version of this package

File: packages/source-readme/src/__tests__/index.test.ts (hunk 3 above)
Comment (mark-tate) [reply packages/source-readme/src/__tests__/index.test.ts:29]: done

File: packages/source-readme/src/__tests__/index.test.ts (hunk 2 above)
Comment (mark-tate) [reply packages/source-readme/src/__tests__/index.test.ts:20]: done

File: packages/source-readme/src/__tests__/index.test.ts (hunk 4 above)
Comment (mark-tate) [reply packages/source-readme/src/__tests__/index.test.ts:37]: done

**Pull request closed without merging**